lib/resty/upstream/healthcheck.lua

```diff
@@ -54,7 +54,6 @@ local function errlog(...)
 end
 
 local function debug(...)
-    -- print("debug mode: ", debug_mode)
```

Comment:
better not introduce unrelated changes.

Reply:
I have reverted these unrelated changes.